spec/lib/bank_accounts/account_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe BankAccount::Account do
+#   # alternate method: 
+#   # it "creates BankAccount::Account.new." do
+#   #   BankAccount::Account.new(0,0)
+#   #   expect(BankAccount::Account).to respond_to(:new).with(2).arguments
+#   # end 
+  context "Creates Account" do
+    let (:new_account) {BankAccount::Account.new(1,555)}
+    let (:negative_account) {BankAccount::Account.new(1,-35)}
```

Comment:
This is something that will be more salient going forward when you are dealing with databases, but you want different accounts to have different ids.